tests/admin_views/tests.py

```diff
@@ -7192,6 +7192,7 @@ def test_save_button(self):
             reverse("admin:auth_user_add"),
             {
                 "username": "newuser",
+                "usable_password": True,
```

Comment:
Following on from my comment earlier, this is relying on `str(True)` rather than an explicit value. As these are values of a query string, I'd expect them to be in serialized form, i.e. `"True"`, that matches the actual value used in the form choices as defined in `SetPasswordMixin`. _(Although maybe `"1"` as for `_continue` below would be more conventional/consistent?)_

Reply:
I'd go for `"true"`; otherwise we have `[value="0"]` and `[value="1"]` as CSS selectors which are not very explicit.